worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/WorldEditPlugin.java

```diff
@@ -575,6 +575,14 @@ BukkitPlayer getCachedPlayer(Player player) {
     }
     //FAWE end
 
+    public BukkitPlayer reCachePlayer(Player player) {
```

Comment:
```suggestion
   BukkitPlayer reCachePlayer(Player player) {
```
Any objections about shifting the modifier? I don't feel this necessarily needs to be public.